distro/src/bin/hz

```diff
@@ -20,19 +20,75 @@ PRGDIR=`cd "$PRGDIR"; pwd`
 # set the parent directory as HAZELCAST_HOME.
 export HAZELCAST_HOME=${HAZELCAST_HOME:-$(cd "$PRGDIR/.." >/dev/null; pwd -P)}
 
-if [ $JAVA_HOME ] ; then
-    RUN_JAVA=$JAVA_HOME/bin/java
-else
-    RUN_JAVA=`command -v java 2>/dev/null`
+# common.sh needs it
+SCRIPT_DIR="$PRGDIR"
+
+. "$PRGDIR"/common.sh
+
+
+if [ "${PROMETHEUS_PORT}" ]; then
```

Comment:
Out of curiosity, I see you omitted this from hz-start:
```shell
# HAZELCAST_CONFIG holds path to the configuration file. The path is relative to the Hazelcast installation (HAZELCAST_HOME).
if [ -z "$HAZELCAST_CONFIG" ]; then
    HAZELCAST_CONFIG="config/hazelcast.xml"
fi
```

Do we support adding custom config with `hz` at all and how?

Reply:
Yeah, it is omitted because we have the following on line 45 of `hz`
```shell
"-Dhazelcast.config=$HAZELCAST_HOME/config/hazelcast.xml" \
```
which is the default config, and if the user wants to provide custom config they can do with
 `hz start -c <hz_config>`

Reply:
Ah, cool, thanks 👍